### PR TITLE
feat(ci): handle the possibility to disable a feature file using @ignore tag

### PR DESCRIPTION
**Pull request by a-eljazouly**
# Description
This pull request introduces the ability to disable specific feature files using tags in Cypress tests. The primary purpose of this change is to provide a more fine-grained control over test execution, allowing us to exclude tests with specific tags, such as `@ignore` for example.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

To verify the changes in this pull request, follow these steps:

1. Ensure that the PR includes a JIRA ticket ID.
2. Check out the code changes and update your local environment.
3. Run Cypress tests using the `cypress-tags` package to exclude tests with the `@ignore` tag.

   ```bash
   cd path_to/tests/e2e
   pnpm run cypress:run --spec ./cypress/e2e/feature_file_path

## Checklist

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/tests/e2e/cypress/e2e/Dashboards/01-dashboard-creation.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@not_exectued
```
Comment (kduret) [inline centreon/tests/e2e/cypress/e2e/Dashboards/01-dashboard-creation.feature:1]:
```suggestion
@not_executed
```

File: centreon/tests/e2e/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "centreon web package",
   "scripts": {
     "cypress:open": "cypress open --config-file cypress.dev.config.ts --browser=chrome --e2e",
-    "cypress:run": "cypress run --config-file cypress.dev.config.ts --browser=chrome --e2e",
+    "cypress:run": "cypress-tags run -e TAGS='not @not_executed --config-file cypress.dev.config.ts --browser=chrome --e2e",
```
Comment (kduret) [inline centreon/tests/e2e/package.json:7]:
```suggestion
    "cypress:run": "cypress-tags run -e TAGS='not @not_executed' --config-file cypress.dev.config.ts --browser=chrome --e2e",
```

File: .github/actions/cypress-e2e-testing/action.yml [hunk 3]
```diff
@@ -55,16 +55,25 @@ runs:
...
+    #     browser: chrome
+    #     install: false
+    #     working-directory: ${{ inputs.module }}/tests/e2e
+    #     spec: cypress/e2e/**/${{ inputs.feature_file_path }}
+    #     publish-summary: false
+
     - name: Cypress end-to-end testing
-      uses: cypress-io/github-action@v6
+      run: |
+        pnpm run cypress:run --spec cypress/e2e/**/${{ inputs.feature_file_path }}
```
Comment (kduret) [inline .github/actions/cypress-e2e-testing/action.yml:74]: you should use this command with cypress action : https://github.com/cypress-io/github-action#custom-test-command

**Review by bdauria — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Review by kduret — APPROVED** [review]
you can merge using "Squash and merge" button

**Pull request merged**